HelmSetup.md

```diff
@@ -112,50 +112,56 @@ After deployed, visit devlake: https://devlake-0.example.com:8443, and grafana a
-| lake.storage.size             | volume size for lake's data                              | 100Mi                      |
-| lake.image.repository         | repository for lake's image                              | apache/devlake             |
-| lake.image.pullPolicy         | pullPolicy for lake's image                              | Always                     |
-| lake.loggingDir               | log dir for the lake server                              | /app/logs                  |
-| lake.loggingLevel             | log level for the lake server                            | info                       |
-| lake.dotenv                   | initial configurations for injecting to lake's .env      | see Values.yaml            |
-| ui.image.repository           | repository for ui's image                                | apache/devlake-config-ui   |
-| ui.image.pullPolicy           | pullPolicy for ui's image                                | Always                     |
-| ui.basicAuth.enabled          | If the basic auth in ui is enabled                       | false                      |
-| ui.basicAuth.user             | The user name for the basic auth                         | "admin"                    |
-| ui.basicAuth.password         | The password for the basic auth                          | "admin"                    |
-| service.type                  | Service type for exposed service                         | NodePort                   |
-| service.uiPort                | Node port for config ui                                  | 32001                      |
-| service.ingress.enabled       | If enable ingress                                        | false                      |
-| service.ingress.enableHttps   | If enable https                                          | false                      |
-| service.ingress.className     | Name for ingressClass. leave empty for using default     | ""                         |
-| service.ingress.hostname      | The hostname/domainname for ingress                      | localhost                  |
-| service.ingress.prefix        | The prefix for endpoints, currently not used             | /                          |
-| service.ingress.tlsSecretName | The secret name for tls's certificate for https          | ""                         |
-| service.ingress.httpPort      | The http port for ingress                                | 80                         |
-| service.ingress.httpsPort     | The https port for ingress                               | 443                        |
-| option.localtime              | The hostpath for mount as /etc/localtime                 | /etc/localtime             |
-| option.database               | The database type, valids: mysql                    | mysql                      |
+| Parameter                         | Description                                              | Default                    |
```

Comment:
I think there are two changes here:

- increase the width of the first column;
- add more lines.

For future PRs, I suggest creating a PR solely for the format adjustment and adding new lines in another PR. Because in that way, it'd be a lot easier to review the change. 

Now it's hard for reviewers to see if the first tens of lines are changed only because of reformat or if the content is also changed. Thanks!

Reply:
Here I added the new values, but some names or descriptions were too long, which forced me to adjust the format to fit in. Sorry about that!